diff -ur kmud2/doc/en/index.docbook kmud2.patched/doc/en/index.docbook
--- kmud2/doc/en/index.docbook	2002-02-04 03:16:33.000000000 +0100
+++ kmud2.patched/doc/en/index.docbook	2004-02-08 12:41:30.000000000 +0100
@@ -2,6 +2,7 @@
 <!DOCTYPE book PUBLIC "-//KDE//DTD DocBook XML V4.1-Based Variant V1.0//EN" "dtd/kdex.dtd" [
   <!ENTITY kmud "<application>Kmud</application>">
   <!ENTITY kappname "&kmud;">
+  <!ENTITY package "kmud">
   <!ENTITY % addindex "IGNORE">
   <!ENTITY % English "INCLUDE"> <!-- change language only here -->
   <!-- Do not define any other entities; instead, use the entities
